docs/integrations/mirroring.md

```diff
@@ -0,0 +1,179 @@
+            for changed_key in parsed_args.delta.keys():
+                old_incident[changed_key] = parsed_args.delta[changed_key]  # type: ignore
+
+            parsed_args.data = old_incident
+
+        else:
+            parsed_args.data['createInvestigation'] = True
+
+        updated_incident = client.update_incident(incident=parsed_args.data)
+        new_incident_id = updated_incident['id']
+        demisto.debug(f'Got back ID [{new_incident_id}]')
+
+    else:
+        demisto.debug(f'Skipping updating remote incident fields [{parsed_args.remote_incident_id}] as it is '
+                      f'not new nor changed.')
+
+    if parsed_args.entries:
+        for entry in parsed_args.entries:
+            demisto.debug(f'Sending entry {entry.get("id")}')
+            client.add_incident_entry(incident_id=new_incident_id, entry=entry)
+
+    # Close incident if relevant
+    if updated_incident and parsed_args.inc_status == IncidentStatus.DONE:
+        demisto.debug(f'Closing remote incident {new_incident_id}')
+        client.close_incident(
+            new_incident_id,
+            updated_incident.get('version'),  # type: ignore
+            parsed_args.data.get('closeReason'),
+            parsed_args.data.get('closeNotes')
+        )
+
+    return new_incident_id
+
+```
+### get-mapping-fields
+* SchemeTypeMapping - this is the object you should use to keep the correct structure of the mapping for the fetched incident type.
+* GetMappingFieldsResponse - this is the object to gather all your SchemeTypeMapping and then to parse them properly into the results using the return_results command.
+Usage example:
+```python
+def get_mapping_fields_command():
+    xdr_incident_type_scheme = SchemeTypeMapping(type_name='incident type example')
+    for field in ['field1', 'field2', 'field3']:
+        xdr_incident_type_scheme.add_field(name=field, description='the description for the field')
+
+    return GetMappingFieldsResponse(xdr_incident_type_scheme)
+
+```
+
+## Incident fields on a XSOAR incident 
+There are few incident fields you need to configure to get going on the system.
+* dbotMirrorDirection - Both or In
+* dbotMirrorId - represents the id of the incident in the external system
+* dbotMirrorInstance - the instance that will perform the mirroring
+
+Useful fields:
+* dbotMirrorTags - tags for mirrored-out entries (comment/files).
+
+## Debugging
+
+* getMirrorStatistics command - a hidden command that returns mirroring statistics: total mirrored, rate limited, and last run.
+* getSyncMirrorRecords - a hidden command that returns records that hold internal mirroring metadata for each mirrored incident
+* get-remote-data - is runnable through the war room with the relevant arguments and see the results.
+* get-mapping-fields - is runnable through the war room with no arguments and see the results.
+
+Also important to note:
+
+DO NOT change dbot mirroring fields for existing incidents after they were set
```

Comment:
need more clarity here
why would someone want to change these fields? what else should they do ?